hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/test/TestOutbox.java

```diff
@@ -131,6 +158,57 @@ public boolean offerToSnapshot(Object key, Object value) {
         return snapshotQueue;
     }
 
+    /**
+     * Move all items from the queue to the {@code target} collection and make
+     * the outbox available to accept more items. Also calls {@link
+     * #resetBatch()}. If you have limited capacity outbox, you need to call
```

Comment:
typo, if you have "a" limited capacity outbox

Reply:
fixed